modules/restoring-the-backend-redis-database.adoc

```diff
@@ -0,0 +1,84 @@
+:_mod-docs-content-type: PROCEDURE
+
+//included in backing-up-and-restoring-3scale-by-using-oadp.adoc assembly
+
+[id="restoring-the-backend-redis-database_{context}"]
+= Restoring the back-end Redis database
+
+You can restore the back-end Redis database by deleting the deployment and specify resorces in a YAMl file, which you do not want to restore.
+
+.Prerequisites
+
+. You restored the `Secret` and `APIManager` custom resources.
+. You restored the `mysql` database.
+
+.Procedure
+
+. Delete the `backend-redis` deployment:
++
+[source,terminal]
+----
+$ oc delete deployment backend-redis -n threescale
+----
++
+.Example output:
++
+[source,terminal]
+----
+Warning: apps.openshift.io/v1 DeploymentConfig is deprecated in v4.14+, unavailable in v4.10000+
+
+deploymentconfig.apps.openshift.io "backend-redis" deleted
+----
+
+. Create a YAML file with the following configuration to restore the Redis database:
++
+.Example restore-backend.yaml file:
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: restore-backend
+  namespace: openshift-adp
+spec:
+  backupName: redis-backup
+  excludedResources:
+    - nodes
+    - events
+    - events.events.k8s.io
+    - backups.velero.io
+    - restores.velero.io
+    - resticrepositories.velero.io
+    - csinodes.storage.k8s.io
+    - volumeattachments.storage.k8s.io
+    - backuprepositories.velero.io
+  itemOperationTimeout: 1h0m0s
+  restorePVs: true
+----
+
+. Restore the Redis database:
++
+[source,terminal]
+----
+$ oc create -f <restore-backend>.yaml
+----
+
+.Verification 
+
+* Verify that the `PodVolumeRestore` restore is completed:
```

Comment:
In an earlier module, you just said this:

```suggestion
* Verify that the `PodVolumeRestore` is completed:
```
Not sure which is correct, but I'd pick one and be consistent between the two places.